utils.go

```diff
@@ -86,3 +88,37 @@ func mapstructureDecode(input interface{}, output interface{}) error {
 
 	return decoder.Decode(input)
 }
+
+// This is to ensure that string value will always be a string representation in Google Sheets.
+// Without this, "1" may be converted automatically into an integer.
+// "2020-01-01" may be converted into a date format.
+func escapeValue(value interface{}) interface{} {
+	switch value.(type) {
+	case string:
+		return fmt.Sprintf("'%s", value)
+	default:
+		return value
+	}
+}
+
+func checkIEEE754SafeInteger(value interface{}) error {
+	switch converted := value.(type) {
+	case int:
+		return isIEEE754SafeInteger(int64(converted))
+	case int64:
+		return isIEEE754SafeInteger(converted)
+	case uint:
+		return isIEEE754SafeInteger(int64(converted))
+	case uint64:
+		return isIEEE754SafeInteger(int64(converted))
```

Comment:
This conversion is safe because when a very big value is converted to int64 and becomes negative, the big (now) negative value will return `false` anyway right?

Reply:
Hmmm yes that's right. But now that I think about it, this is not correct also, right? If we have a value that is 2 * 2^53 (which is the limit of IEEE 754 before losing precision), then the `int64` version might make it within the range after shifting. However, since there is no unsigned concept in floating point system, I believe 2 * 2^53 is still above the max limit before precision lost.

What do you think?

Reply:
![image](https://user-images.githubusercontent.com/6941596/192670129-13031bb6-ba34-44d7-9495-6191ae614158.png)

For the first one, it's still within the 2^53 upper boundary. The next one is 2^53+1, the last one is 2^53+2. You can see the second one has a precision lost on the final digit before the exponent part while the last one is multiply of 2.

Reply:
> If we have a value that is 2 * 2^53 (which is the limit of IEEE 754 before losing precision), then the int64 version might make it within the range after shifting

Oh hmm that is possible. Probably the easier way is to have 2 function to check the range, one for unsigned and one for the signed one.

Wdyt?

Reply:
@fatanugraha updated.